react/react1/week1/vite-project/src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./App.css";
+import { Header } from "./Header.jsx";
+
+function App() {
+  return (
+    <div className="App">
+      <Header></Header>
```

Comment:
You can just add the closing tag directly to your component like this: `<Header />`

Reply:
Yes, totally forgot about it 😅
Thank you for review:)